scipy/stats/_stats_py.py

```diff
@@ -10797,66 +10797,87 @@ def rankdata(a, method='average', *, axis=None, nan_policy='propagate'):
     array([ 2.,  3.,  4., nan,  1., nan])
 
     """
-    if method not in ('average', 'min', 'max', 'dense', 'ordinal'):
+    method = str(method).lower()
```

Comment:
Should capital method name (MIN, MAX, etc) be supported?

Reply:
Sure, I can make that change.

Reply:
I’m not sure other stats module’s APIs also supports capital method name or a plan for it. 
If not or no plan, I think we should only support exact method name only for API consistency.

Reply:
It's inconsistent, but I think most functions require the correct case. It look like the `rankdata` `method` parameter was strict about case before this PR, and `nan_policy` is typically (if not always) strict about case, so I can remove this line to follow that precedent.